lib/matrix_api_lite/generated/model.dart

```diff
@@ -82,6 +106,20 @@ class DiscoveryInformation {
   IdentityServerInformation? mIdentityServer;
 
   Map<String, Map<String, Object?>> additionalProperties;
+
+  @override
+  bool operator ==(Object other) =>
+      other is DiscoveryInformation &&
+      other.mHomeserver == mHomeserver &&
+      other.mIdentityServer == mIdentityServer;
+
+  @override
+  int get hashCode => mHomeserver.hashCode ^ mIdentityServer.hashCode;
```

Comment:
Pretty sure this should be

```suggestion
  int get hashCode => Object.hash(mHomeserver, mIdentityServer);
```

Reply:
done